website/docs/users-sources/sources/social-logins/google/workspace/index.md

```diff
@@ -0,0 +1,197 @@
+
+### Creating a Federation Source
+
+From the admin interface, navigate to _Directory_ › _Federation & Social login_ and press **Create**.
+
+Within the new source dialog, choose **SAML Source** and continue by filling in the following fields with your desired values:
+
+| Field | Value            |
+| ----- | ---------------- |
+| Name  | Google Workspace |
+| Slug  | `google`         |
+
+:::info
+
+Heads up! Your choice of `slug` should match the ACS URL you provided to Google Workspace.
+You can choose a different slug, but you will need to update the ACS URL in Google Workspace to match.
+
+:::
+
+#### Protocol Settings
+
+Next, we'll configure the SAML protocol settings for the source. Fill in the following fields with the values you copied from Google Workspace:
+
+|                  |                                                           |
+| ---------------- | --------------------------------------------------------- |
+| SSO URL          | `https://accounts.google.com/o/saml2/idp?idpid=#########` |
+| Entity ID/Issuer | `https://authentik.company`                               |
+
+#### Advanced Protocol Settings
+
+Depending on your Google Workspace configuration, you may need to adjust the advanced protocol settings.
+
+| Field                     | Value         |
+| ------------------------- | ------------- |
+| Allow idP-initiated Login | Enabled ✅    |
+| NameID Policy             | Email Address |
+
+Finally, save the source configuration and confirm the app is present in the list of federated sources.
+
+## Testing your configuration
+
+To test your configuration, navigate to the login page of your authentik instance and confirm the Google Workspace option is available as an alternative login method.
+
+Next, click on the Google Workspace button and confirm that you are redirected to authenticate via your Google Workspace credentials. After successful authentication **with a non-super-admin account**, you should be redirected back to your authentik instance and logged in.
+
+## Troubleshooting
+
+Most issues stem from a misconfiguration on Google Workspace or authentik. However, your workspace may take a few minutes to propagate changes depending on the size of your organization.
+
+### `403 app_not_configured_for_user`
+
+Confirm that the entity ID (AKA "Issuer") matches the value you've provided both in Google Workspace and authentik. This can be any unique identifier, but it must match between the two services.
+
+### `403 app_not_enabled_for_user`
+
+In the Google Workspace Admin Console, go to Menu › Apps › Web and mobile apps.
+
+1. In the app list, locate the SAML app generating the error.
+2. Click the app to open its Settings page.
+3. Click User access.
+4. Turn the app ON for everyone or for the user’s organization.
+
+This may take a few minutes to propagate, so try logging in again after a short wait.
+
+## External links
```

Comment:
I think I like this new addition... though as I mentioned, it creates a maintenance load to verify these links don;t change, and to add this section to the other docs. Let's leave it and think on the ROI. ;-)

Reply:
External references...